### DISCUSSION TITLE
apt error fetching regolith repository

### DISCUSSION THREAD

**Discussion by simoncasassus** [regolith-linux/regolith-desktop#1035] [General]
i am installing Regolith 3.2 on a new machine, and got this error message trying to run sudo apt update:
Get:6 https://regolith-desktop.org/testing-ubuntu-noble-amd64 noble InRelease [2,376 B]
Err:6 https://regolith-desktop.org/testing-ubuntu-noble-amd64 noble InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 7107DED133505B88

**Comment by kgilmer** [comment]
Hi @simoncasassus , hmm I'm not seeing any issues in [automation](https://github.com/regolith-linux/voulage/actions/runs/10048748538).  Did you install the Regolith public key as describe in the [installation steps](https://regolith-desktop.com/docs/using-regolith/install/)?

**Comment by simoncasassus** [comment]
hi Ken, thanks - sorry about that.
